cmd/wipeout.go

```diff
@@ -35,10 +35,33 @@ func wipeoutBase() CommandComponent {
 func makeWipeoutCmd(ctx context.Context, app *AppComponents) *cobra.Command {
 	cmp := Compose(app.Global, wipeoutBase(), app.Wipeout)
 	cmd := &cobra.Command{
-		Use:               "wipeout [flags]",
-		Long:              `Destroys all managed keys and certificates.`,
+		Use: "wipeout [flags] ['ca'|'keys']",
+		Long: `Destroys all managed keys and certificates.
+
+If given literal arguments 'ca' or 'keys', then just wipes out certs or keys respectively.
+`,
 		PersistentPreRunE: cmp.PersistentPreRunE,
-		RunE:              ComposeRun(cmp, rotate.Wipeout),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			if cmp != nil {
```

Comment:
should this be cmp == nil?

Reply:
No. I've collapsed the logic from ComposeRun a little better now.